pkgs/servers/varnish/default.nix

```
@@ -12,6 +12,8 @@ let
         inherit sha256;
       };
 
+      patches = extraPatches;
```

do we need the `extra` if we're going to overwrite everything anyway?
```suggestion
      inherit patches;
```

---

I guess this is a backport, so the time to comment has already passed, and it's a  subjective change

---

Yeeeeah I was just imagining it becoming unclear in the future if there were some base patches we had to put on all versions... and then it wouldn't be clear whether the list added there would overwrite or extend... so we'd have to rename it and then people out of tree might be overriding it with the old arg name and yknow...